WebRoot/css/zm.css

```diff
@@ -4109,40 +4109,80 @@ HTML>BODY .appt-selected .appt_allday_body
 	vertical-align:middle;
 }
 
-.ZmFilterRuleDialog .FilterRuleSection,
-.ZmFilterRuleDialog .FilterActionSection {
-	background-color:rgba(255,255,255,.5);
-	border:1px solid rgba(0,0,0,.25);
-	margin:.25em 1em 1em;
-	padding:0px;
-#IFDEF MSIE
+.ZmFilterRuleDialog .WindowOuterContainer {
+	@FilterRuleDialogOuterContainer@
+}
+
+.ZmFilterRuleDialog .ZmFilterNameSection {
+	@FilterRuleDialogFilterNameSection@
+}
+
+.ZmFilterRuleDialog .ZmFilterName {
+	@FilterRuleDialogFilterName@
+}
+
+.ZmFilterRuleDialog .ZmFilterActiveLabel {
+	@FilterRuleDialogCheckBoxLabel@
+}
+
+.ZmFilterRuleDialog .ZmFilterNameInput {
+	@FilterRuleDialogInputWrapper@
+}
+
+.ZmFilterRuleDialog .ZmFilterRuleSection,
+.ZmFilterRuleDialog .ZmFilterActionSection {
+	@FilterRuleDialogFieldSet@
+	#IFDEF MSIE
```

Comment:
I am not sure, but do we need this now? Have we tested on IE, if it's really needed?